<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SW_switchClass</class>
 <widget class="QMainWindow" name="SW_switchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SW_switch</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stats</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="port_1">
       <row>
        <property name="text">
         <string>EthernetII</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>ARP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>IP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>UDP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>HTTP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>ICMP</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>IN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>OUT</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="port_2">
       <row>
        <property name="text">
         <string>EthernetII</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>ARP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>IP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>TCP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>UDP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>HTTP</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>ICMP</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>IN</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>OUT</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="port_1_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="port_2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearStats">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="camTable">
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>5</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Port</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>MAC</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Timer</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearCamTable">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR TABLE</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="timer">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>380</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer:</string>
    </property>
   </widget>
  </widget>
  <action name="actionCAM_TABLE">
   <property name="text">
    <string>CAM TABLE</string>
   </property>
  </action>
  <action name="actionCAM">
   <property name="text">
    <string>CAM</string>
   </property>
  </action>
  <action name="actionSTATISTICS">
   <property name="text">
    <string>STATISTICS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sw_switch.qrc"/>
 </resources>
 <connections/>
</ui>
